libs/api-client-bear/package.json

```diff
@@ -94,6 +94,7 @@
         "ts-loader": "^8.3.0",
         "typescript": "4.0.2",
         "webpack": "^5.58.0",
-        "webpack-cli": "^4.9.2"
+        "webpack-cli": "^4.9.2",
+        "blob-polyfill": "^7.0.20220408"
```

Comment:
For which browsers do wee need this?

Reply:
This is not require by browser but by jest suite. Some tests ends up calling the export functions where `Blob` is created. The test suite uses `node` environment but `Blob` is available only in `jsdom`.